### PR TITLE
MNTOR-2934/upload sentry sourcemaps for Docker deploy builds

### PR DESCRIPTION
**Pull request by rhelmer**
# References: 
Jira: MNTOR-2934

# Description

Always upload Sentry sourcemaps when performing Docker deploy builds.

This was restricted to only builds triggered by tag before, now that we've moved over to deploying from `main` and do the Docker push from GH actions, we can simplify things.

# Checklist (Definition of Done)
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.

### CODE REVIEW

File: .github/workflows/docker_build_deploy.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@ jobs:
           echo "{\"commit\":\"$GITHUB_SHA\",\"version\":\"$GITHUB_REF\",\"source\":\"https://github.com/$GITHUB_REPOSITORY\",\"build\":\"$GITHUB_RUN_ID\"}" > version.json
 
       - name: Build Docker image
+        env:
+          UPLOAD_SENTRY_SOURCEMAPS: true
```
Comment (mansaj) [inline .github/workflows/docker_build_deploy.yml:35]: Looks like we no longer need the env var `UPLOAD_SENTRY_SOURCEMAPS`?

I believe we can remove it from 
- `.env-dist`
- `next.config.js`

Comment (rhelmer) [reply .github/workflows/docker_build_deploy.yml:35]: I think it still needs to be set as a variable, we need some way to get it into `next.config.js` right? I'm just setting it in the yaml vs. setting it in the GHA environment and then passing it through the yaml.

We only want it to run in this context, not on every PR and every time someone might run `npm run build` locally right?

Comment (mansaj) [reply .github/workflows/docker_build_deploy.yml:35]: Ah hmm, I think I misunderstood what's going on here. NVM!

**Review by mansaj — APPROVED** [review]

File: .github/workflows/docker_build_deploy.yml (hunk 1 above)
Comment (rhelmer) [reply .github/workflows/docker_build_deploy.yml:35]: np, thanks!

**Pull request merged**